### PR TITLE
Add Skyscanner to adopters lists

### PR DESCRIPTION
**Pull request by danielgblanco**
This change:

- Adds Skyscanner to the adopters list
- Sorts the list of adopters alphabetically. It seems it was sorted alphabetically and then at some point stopped being sorted alphabetically. Happy to revert that commit if it's not the intention of this list to be sorted alphabetically.

### CODE REVIEW

File: .cspell.yml [hunk 1]
```diff
@@ -144,6 +144,7 @@ words:
   - Sharma
   - shippingservice
   - shortcode
+  - Skyscanner
```
Comment (svrnm) [inline .cspell.yml:147]: put Skycanner in line1 of the adopters.yaml, as long as this is not used across multiple pages we keep this local

**Comment by svrnm** [comment]
Nice! Especially that this comes with a reference :-)

File: .cspell.yml (hunk 1 above)
Comment (danielgblanco) [reply .cspell.yml:147]: Ah! I see! I completely missed the cspell comment. Will do now.

**Review by svrnm — APPROVED** [review]

**Pull request merged**